README.md

```diff
@@ -71,20 +71,11 @@ To connect it to slack you need to meet this preconditions:
 > Once your bot is ready for production you should update your commands url to a permanent one.
 > [Heroku](https://duckduckgo.com/?q=flask+on+heroku&t=brave&ia=web) might be a good choice if you are just getting started as it has a generous free tier.
 
-### Does it support new slack Shorcuts?
-Yes, See [examples/shortcuts.py](examples/shortcuts.py) for a self contained example
-
-### And can i use new slack Modals?
-Of course! See [examples/views.py](examples/views.py) for a quick example
-
-### Are interactive actions supported?
-Yes! See [examples/actions.py](examples/actions.py) for a quickstart.
+### Slackify supports Shortcuts, Slash Commands, Events, Actions, Modals, and Block Kit. See the below examples:
+Check the examples directory above for all examples.
 >Note: Legacy actions are unsupported by design as they are discouraged by slack. Nevertheless, if there's popular demand, we could add support for them.
 
-### And slack events?
-As you may have guessed, they are also supported. See [examples/events.py](examples/events.py) for an example.
-
-## Full example
+## Full example (Also in above examples directory)
```

Comment:
I would not clutter the title with this info, perhaps a note at the beginning/end of the example that links to the file on the repo

Reply:
Will do. I'll revert